corehq/apps/user_importer/importer.py

```diff
@@ -687,6 +668,31 @@ def create_or_update_commcare_users_and_groups(upload_domain, user_specs, upload
     return ret
 
 
+def _setup_commcare_user_for_import(domain, user_id, username, is_account_confirmed, web_user, password,
```

Comment:
How about "get_or_create" to be more specific than "setup"?

Reply:
https://github.com/dimagi/commcare-hq/pull/29935/commits/44c4898e0c54d7786ae50ca44381a9f6a8ab2ecf

a catch to that is that the user is actually not "created" because the record is not saved yet. Was thinking of `get_or_init` though that would be confusing as well.

Reply:
That's a good point. `get_or_init` sounds fine, too.